src/components/SystemMessageList/SystemMessageList.tsx

```diff
@@ -0,0 +1,54 @@
+import React, {useEffect, useState} from 'react';
+import {SystemMessageNotification, SystemMessageNotificationTypes} from '../SystemMessageNotification/SystemMessageNotification';
+import {getSystemMessages} from '../../api';
+import {useMount} from '../../hooks/useMount';
+
+
+export interface SystemMessageList {
+  id: number;
+  type: SystemMessageNotificationTypes;
+  description: string;
+
+}
+
+const SystemMessagesList = () => {
+
+  const [systemMessageList, setSystemMessageList] = useState<SystemMessageList[]>([]);
+
+  const [dismissedMessage, setDismissedMessage] = useState<string[]>([]);
+
+  useEffect(() => {
+    window.localStorage.setItem('dismissedMessage', dismissedMessage.join(','));
+  }, ['dismissedMessage']);
+
+  useMount(async () => {
+    const messages = await getSystemMessages();
+    setSystemMessageList(messages);
+    const verifySessionID = window.localStorage.getItem ? ('dismissedMessage').split(',') : null;
+    if (verifySessionID) {
+      setDismissedMessage(verifySessionID);
+    }
```

Comment:
Once we have the dismissed messages from local storage, we need to search the messages and set the current active message to the first one we find that isn't dismissed. We can use [Array.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) for this.

```
setCurrentMessage(messages.find(m: SystemMessageList => !verifySessionID.includes(m.id.toString())));
```